### PR TITLE
fix(tests) :Added detail page tests

### PR DESCRIPTION
**Pull request by Raunak1203**
<strong>Note: If there are pending changes to the PR, prefix the PR title with "WIP" and add the label "DO NOT MERGE"</strong>

### Mandatory
 - [x] What does this PR do? 
This PR adds tests for detail page
- [x] What issue/task does this PR references?
https://github.com/fabric8-ui/fabric8-planner/issues/2624
- [x] Are the tests Included?
Yes
___
### Optional
- [ ] Is the documentation Included?

- [ ] Are the Release Notes included?

- [ ] @mention(s) to expected reviewer(s) included

### CODE REVIEW

**Comment by Raunak1203** [comment]
[test]

**Comment by Raunak1203** [comment]
[test]

**Comment by Raunak1203** [comment]
[test]

**Comment by Raunak1203** [comment]
[test]

**Comment by Raunak1203** [comment]
[test]

File: src/tests/specs/detailPage.spec.ts [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+      title: "detail page title updated",
+      description: 'New WorkItem Description'
+    };
+    await planner.detailPage.updateTitle(updatedWorkItem.title);
+    expect(await planner.detailPage.titleInput.getAttribute('value')).toBe(updatedWorkItem.title);
+    await planner.detailPage.updateDescription(updatedWorkItem.description);
+    expect(await planner.detailPage.getDescription()).toBe(updatedWorkItem.description);
+  });
+    
+  it('Associate workitem with an Area', async () => {
```
Comment (rgarg1) [inline src/tests/specs/detailPage.spec.ts]: Pls use the syntax that starts with _should_

Comment (Raunak1203) [reply src/tests/specs/detailPage.spec.ts]: sure @rgarg1

**Comment by Raunak1203** [comment]
[test]

**Comment by jarifibrahim** [comment]
[test]

**Comment by Raunak1203** [comment]
@rgarg i have added test for workitem state. Please check

File: src/tests/specs/detailPage.spec.ts [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    await planner.quickPreview.notificationToast.untilHidden();
+  });
+  
+  it('should open detail view and apply label', async () => {
+    let workitemname = {"title": "detail page test"},
+      label = 'Label1';
+    await planner.createWorkItem(workitemname);
+    await planner.workItemList.openDetailPage(workitemname.title);
+    await planner.waitUntilUrlContains('detail');
+    await planner.detailPage.addLabel(c.label);
```
Comment (jarifibrahim) [inline src/tests/specs/detailPage.spec.ts]: This should be `addLabel(label1)`

File: src/tests/ui/planner/workitem-detailpage.ts [hunk 3]
```diff
@@ -2,6 +2,13 @@ import * as ui from '../../ui';
 import { WorkItemQuickPreview } from './workitem-quickpreview';
 
 export class WorkItemDetailPage extends WorkItemQuickPreview {
+
```
Comment (jarifibrahim) [inline src/tests/ui/planner/workitem-detailpage.ts]: Remove this, please.

**Comment by Raunak1203** [comment]
[test]

**Comment by Raunak1203** [comment]
[test]

**Pull request merged**